cluster-api/src/main/resources/application.properties

```diff
@@ -10,30 +9,39 @@ server.port=9343
 
 # SSL properties to connect to Kafka clusters
 
-klawssl.kafkassl.keystore.location=client.keystore.p12
-klawssl.kafkassl.keystore.pwd=klaw1234
-klawssl.kafkassl.key.pwd=klaw1234
-klawssl.kafkassl.truststore.location=client.truststore.jks
-klawssl.kafkassl.truststore.pwd=klaw1234
-klawssl.kafkassl.keystore.type=pkcs12
-klawssl.kafkassl.truststore.type=JKS
+#clusterid.kafkassl.keystore.location=client.keystore.p12
+#clusterid.kafkassl.keystore.pwd=klaw1234
+#clusterid.kafkassl.key.pwd=klaw1234
+#clusterid.kafkassl.truststore.location=client.truststore.jks
+#clusterid.kafkassl.truststore.pwd=klaw1234
+#clusterid.kafkassl.keystore.type=pkcs12
+#clusterid.kafkassl.truststore.type=JKS
 
 # SASL properties to connect to Kafka clusters
-#acc1.kafkasasl.jaasconfig.plain=org.apache.kafka.common.security.plain.PlainLoginModule required username='kwuser' password='kwuser-secret';
-#acc1.kafkasasl.jaasconfig.scram=org.apache.kafka.common.security.scram.ScramLoginModule required username='kwuser' password='kwuser-secret';
-#acc1.kafkasasl.saslmechanism.gssapi.servicename=kafka
-#acc1.kafkasasl.jaasconfig.gssapi=com.sun.security.auth.module.Krb5LoginModule required useKeyTab=true storeKey=true keyTab="/location/kafka_client.keytab" principal="kafkaclient1@EXAMPLE.COM";
+#clusterid.kafkasasl.jaasconfig.plain=org.apache.kafka.common.security.plain.PlainLoginModule required username='kwuser' password='kwuser-secret';
+#clusterid.kafkasasl.jaasconfig.scram=org.apache.kafka.common.security.scram.ScramLoginModule required username='kwuser' password='kwuser-secret';
+#clusterid.kafkasasl.saslmechanism.gssapi.servicename=kafka
+#clusterid.kafkasasl.jaasconfig.gssapi=com.sun.security.auth.module.Krb5LoginModule required useKeyTab=true storeKey=true keyTab="/location/kafka_client.keytab" principal="kafkaclient1@EXAMPLE.COM";
 
-kafkasasl.saslmechanism.plain=PLAIN
-kafkasasl.saslmechanism.gssapi=GSSAPI
-kafkasasl.saslmechanism.scram.256=SCRAM-SHA-256
-kafkasasl.saslmechanism.scram.512=SCRAM-SHA-512
+# Schema Registry/karapace HTTPS credentials (Ex : username:password)
+#clusterid.klaw.schemaregistry.credentials=
 
-# User for accessing Cluster api
-klaw.clusterapi.access.username=kwclusterapiuser
-# Provide a base 64 encoded string below. The same secret should be configured in Klaw Api. Change to a new one. Ex : dGhpcyBpcyBhIHNlY3JldCB0byBhY2Nlc3MgY2x1c3RlcmFwaQ==
+# Kafka Connect HTTPS credentials (Ex : username:password)
+#clusterid.klaw.kafkaconnect.credentials=
```

Comment:
Small UX comment, I think it's good to add some comment on each block saying uncomment this section for SSL or SASL or the corresponding type.